.github/workflows/gh-ci.yaml

```diff
@@ -9,6 +9,10 @@ on:
       - develop
       - master
 
+concurrency: 
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
```

Comment:
You directly followed https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-using-concurrency-to-cancel-any-in-progress-job-or-run so I assume this will work.

But I just want to check: this will only cancel jobs related to _this_ PR, not anything in other PRs, right?

Reply:
Yup, just in case though I'll cycle the CI again just to make sure it doesn't kill the currently running H5MD workflow.

Reply:
![image](https://user-images.githubusercontent.com/12460125/118191589-b86b2880-b43c-11eb-801d-c8a025164fba.png)

Seems to work fine :)